client/state/account/actions.js

```diff
@@ -15,7 +15,7 @@ export function closeAccount() {
 
 export function closeAccountSuccess() {
 	return async ( dispatch ) => {
-		await user().clear();
+		await clearStore();
```

Comment:
I think this action doesn't need to call `clearStore` at all 🙂 The `AccountSettingsClose` already does that in a lifecycle method when the closing is finished, by calling `redirectToLogout`:

https://github.com/Automattic/wp-calypso/blob/trunk/client/me/account-close/main.jsx#L52

We're currently clearing the store twice.

Reply:
Good call. I've cleaned this up in 90a76101564fbeb140838284da57c1ccb2f3a1d2